site/Dockerfile

```diff
@@ -1,3 +1,47 @@
-FROM klakegg/hugo:ext-alpine
+# Credit to Julien Guyomard (https://github.com/jguyomard). This Dockerfile
+# is essentially based on his Dockerfile at
+# https://github.com/jguyomard/docker-hugo/blob/master/Dockerfile. The only significant
+# change is that the Hugo version is now an overridable argument rather than a fixed
+# environment variable.
```

Comment:
This could also have a mention to being based on the k/website one. WDYT?

Reply:
Added credit to the k/website.